### ISSUE TITLE
(chore) install two flags on cloudflare workers

### ISSUE THREAD

**Issue by sirpy** [GoodDollar/GoodProtocolUI#145]
follow instructions here: https://github.com/twoflags-io/twoflags-api
resolver domain: twoflags-resolver.goodworker.workers.dev
api domain: twoflags-api.goodworker.workers.dev

**no need for zone_id/route in toml config file use above domains**
- [ ] using wrangler cli install on our cloudflare, creating necessary kv stores and all configurations
- [ ] demo the API on swagger editor (they use openapi specs)

**Comment by DmitryIlchenko** [comment]
@sirpy 
We figured it out but we couldn’t deploy because the problem is on the side of the code.

We can work with Cloudflare but we cannot deploy this code due to an unknown error on the code side. If you need to quickly do something and deploy it for this task, please provide us with access to Cloudflare.

**Comment by DmitryIlchenko** [comment]
This task took longer than we expected because you didn't give access to Cloudflare. We don't understand why you didn't do it.

1. Here is the list of endpoints:
GET-/environments
POST-/environments
PATCH-/environments
DELETE-/environments
GET-/apikeys
POST-/apikeys
DELETE-/apikeys
GET-/flags
POST-/flags
DELETE-/flags
PATCH-/flags
POST-/flags/selectors
GET-/flags/selectors
DELETE-/flags/selectors
GET-/namespaces
POST-/namespaces
PATCH-/namespaces
DELETE-/namespaces
POST-/accounts
GET-/values
PATCH-/values

To make a request on them, the headers must contain Authorization with Bearer 9d443813-5d6e-43f5-9a38-77f81752be13

The main link - [woflags-api.goodworker.workers.dev](http://twoflags-api.goodworker.workers.dev/)

Everything is deployed on domains.

2. We didn`t understand why this link is needed - twoflags-resolver.goodworker.workers.dev . We didn't find it in the documentation nor in the code.

Example:
Request 
<img width="1508" alt="image" src="https://user-images.githubusercontent.com/74401645/168790083-2036448b-1c6e-48b7-9192-64a0e1d7e9ce.png">

Answer
<img width="1251" alt="image" src="https://user-images.githubusercontent.com/74401645/168790082-430a029d-b5bb-476b-8ddc-64188e9eeb0a.png">

**Comment by sirpy** [comment]
@DmitryIlchenko please have the devops create a repo with the two-flags code + the configuration files
notice not to commit any secrets directly in the configuration files but use env variables

**Comment by sirpy** [comment]
@johnsmith-gooddollar we have feature flags service we can use...

**Comment by DmitryIlchenko** [comment]
@sirpy here is the repo https://github.com/GoodDollar/two-flags

Check it, please.

**Comment by sirpy** [comment]
The repo should be forked from the original two flags, why is the README and INSTALL missing?

**Comment by DmitryIlchenko** [comment]
@sirpy 
README and INSTALL added. The fork itself makes no sense to do. You want to make some edits and so on. If we make a fork nothing will work, the original code from this https://github.com/twoflags-io/twoflags-api does not work as it should.

Do you really need a fork after we added readme and install?

**Comment by sirpy** [comment]
@DmitryIlchenko yes, all changes should be applied on the original fork, so changes that have been done are easy to see.

**Comment by DmitryIlchenko** [comment]
@sirpy 
pushed the separate branch https://github.com/GoodDollar/twoflags-api/tree/workable-stage
You can merge to master too.

Check it

**Comment by johnsmith-gooddollar** [comment]
@DmitryIlchenko i've merged to **our** master to check actions. Now deployment fails (but it worked before)
please check. 

about other stuff - everything is fine, good job, thanks!

**Comment by DmitryIlchenko** [comment]
@johnsmith-gooddollar Tell me did you delete wrangler.toml on purpose or by accident?

**Comment by johnsmith-gooddollar** [comment]
@DmitryIlchenko i just merged your branch to master by creating PR. maybe it was removed by the merge

**Comment by sirpy** [comment]
@DmitryIlchenko this is very simple.
1. delete the repo you did
2. fork two flags repo
3. dev should apply his local changes/commits to the fork

please consult your CTO on how to do this properly

**Comment by DmitryIlchenko** [comment]
1. We don`t have permission to delete this repository - https://github.com/GoodDollar/two-flags
2. Points 2 and 3 have already been done. Result is here - https://github.com/GoodDollar/twoflags-api

@sirpy

**Comment by sirpy** [comment]
@DmitryIlchenko thanks

**Comment by sirpy** [comment]
@DmitryIlchenko 
https://developers.cloudflare.com/workers/learning/logging-workers/#view-logs-using-wrangler-tail
